content/blog/project-management-for-individuals/index.md

```diff
@@ -0,0 +1,134 @@
+
+5. Refactoring / Documenting: Now that it's implemented and working, I do some refactoring and documenting where needed to make it easier to understand.
+
+6. CI: Now I will push all of my commits, and wait for the Continuous Integration pipeline to verify that all the testing, linting and building is successful
+
+7. (Optional) CD: This step is optional because I don't always deploy the changes I make immediately. If I'm fixing an important bug then this step will take place immediately after CI. If not then I'll wait to accumulate a few features to deploy my updated application
+
+8. Start again
+
+## How do you come up with your own methodology?
+
+This is an incredibly complex question, because you can make this as simple or as complicated as you like, and it might change depending on the tools you're using or the technologies you work with.
+
+But I still wanted to put out a general sort of guide to coming up with your own methodology that can be applicable to your software projects, so here are my tips:
+
+### 1. How do you want to break down your to-dos?
+
+In my case, I wanted to break them down into Tasks and Bugs, each with optional tags. And it works great for me because it integrates well with Azure DevOps. But if you're using a different system then maybe you want to rethink this. Maybe you want Test Writing to be it's own type of task because you find it feels better in your system. It's *your* methodology so remember to tailor it to yourself.
+
+### 2. Which branching strategy do you want?
+
+There are many options. I like to go for a simple branching strategy as it's just me working on this, so I don't need anything overly complex, but maybe if you want more structure you can be more complicated. I like to have a branch per issue, then it all gets added to the same "nightly" branch which happens to be my default / main branch, and a production branch that triggers my CD.
+
+### 3. What will your development cycle look like?
+
+Whether it's well defined or not defined at all you inevitable have some sort of routine that you follow when programming. Write this down and also maybe consider if there are any improvements you could easily add to your routine. If you have trouble remembering when to commit for example (like me!) then try to set some sort time when you commit, like for example, when you've written 5 functions or when you've finished a step in your development cycle.
+
+### 4. Miscellaneous bits
+
+There are many other things that you can add to your methodology, but I chose to skip in mine. For example, a priority system, the fields you want your task types to have, do you want an effort system?, do you want to write a word document for a major feature? do you want to include notes in the task?
+
+You can also work out how your pipelines are going to work. Do you want CI to be separate from your CD? Do you want the CI/CD pipeline run automatically or manually?
```

Comment:
```suggestion
You can also work out how your pipelines are going to work. Do you want Continuous Integration (CI) to be separate from your Continuous Delivery (CD)? Do you want the CI/CD pipeline run automatically or manually?
```

I think the difference between CI and CD is subtle enough for many developers that saying the whole word once here would help some readers continue learning about the difference.

Props for adding the distinction between them here too! I love it.